cms/articles/models.py

```diff
@@ -474,3 +483,39 @@ def get_context(self, request: "HttpRequest", *args: Any, **kwargs: Any) -> dict
         context["has_notices"] = bool(notices)
 
         return context
+
+    def ld_entity(self) -> dict[str, object]:
+        """Add statistical article specific schema properties to JSON LD."""
+        # TODO pass through request to this, once wagtailschemaorg supports it
+        properties = {
+            "url": self.get_full_url(),
+            "headline": self.seo_title or self.listing_title or self.get_full_display_title(),
+            "description": self.search_description or self.listing_summary or strip_tags(self.summary),
+            "datePublished": self.first_published_at.isoformat() if self.first_published_at else None,
```

Comment:
I don't think `first_published_at` is the appropriate value based on the context and how we want search engines to interpret this information. This approach won't work in situations such as unpublishing the page or migrating an old release from Florence to Wagtail. In these cases, `first_published_at` will be later than `release_date` since we're migrating a past release. I suggest we stick with `release_date` and convert it to a full ISO datetime format, as the requirements documentation specifies the value uses `T00:00:00.000Z` for the time part.

Reply:
Good points, but I'd have concerns about using `release_date` converted to datetime too, does that create potential issues for articles with a legally required release time if certain metadata like this would then contradict that, showing `00:00:00.000Z` instead of the correct/legally required publish time?

Reply:
After a chat with @zerolab we've decided that reverting to the `release_date` is the best solution for the moment ([36a71b1](https://github.com/ONSdigital/dis-wagtail/pull/250/commits/36a71b16c55e00f7d48b7e76221bc9e862b34865)). Google's validation does raise this as a non-critical issue but this seems to be more of an "opportunity to optionally include more specific data" rather than an issue with the data we're providing, and `release_date` is the most precise and correct field we currently have for this.

Reply:
Agreed 👍